### DISCUSSION TITLE
"strange" text layout of the web interface

### DISCUSSION THREAD

**Discussion by clemente3905** [hassio-addons/addon-ssh#443] [Q&A]
Hi, I usually use the web interface of the Terminal from inside HA.
From a couple of updates the text is acting wierd (see the screenshot, mainly seems a wrong zoom) whilst if I connect via ssh everything is fine.
![image](https://user-images.githubusercontent.com/75305208/191300762-0f75a184-6ce1-47b6-8d7c-dcccf49b8356.png)

Is there any way to restore the standard chars zoom (or to change web interface width X height) in the web interface?

Thanks in advance for help.

**Comment by ac0mputerguru** [comment]
Same here. It goes into 193x49 mode and parts of every letter are missing. I can usually make out what the text is supposed to be, but it's VERY annoying. How can I change the resolution?
